azerothcore/azerothcore-wotlk
# fix(DB): Draenei Start Area Class Training Quests Fixes

## Changes Proposed:
Fix three different issues with these quest:
-  Quest should require Replenishing the Healing Crystals completed
-  Quest should be can auto-complete when talk to the trainer and see quest complete text; no need for accept then complete - this is why no LogDescription or QuestDescription can be find for these quests
- Quests should all only be available to Draenei. Before was inconsistent.

## Issues Addressed:

- Closes #9922 

## SOURCE:

https://tbc.wowhead.com/quest=9290/mage-training
https://tbc.wowhead.com/quest=9288/hunter-training
https://tbc.wowhead.com/quest=9421/shaman-training
https://tbc.wowhead.com/quest=9291/priest-training
https://tbc.wowhead.com/quest=9421/shaman-training
https://tbc.wowhead.com/quest=9289/warrior-training

## Tests Performed:

- Created character, quest is not present. Complete quest 9280, quest is present with question mark icon. Completion text is correctly show. Test with non-Draenei character, quest is not available.


## How to Test the Changes:


1. Create Draenei character, level to 2
2. Visit trainer, see quest is not available
3. Complete quest 9280, quest is now available
4. Quest text is good completion text, no blank Accept text box.

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
